### PR TITLE
added resumes, cover letters tabs

### PR DESCRIPTION
**Pull request by jomanozgml**
- titles only
- next to the vacancies
2023-01-11_15h43_23

### CODE REVIEW

**Review by kovarny — CHANGES_REQUESTED** [review]
I think we talked about pages too, not just tabs. Can you create dummy pages with simple text boxes in them as well?

**Comment by jomanozgml** [comment]
changes implemented
2023-01-12_16h01_35

File: lib/search/search_page.dart [hunk 1]
```diff
@@ -109,6 +109,69 @@ class VacanciesPage extends ConsumerWidget {
   }
 }
 
+class ResumesPage extends ConsumerWidget {
```
Comment (kovarny) [inline lib/search/search_page.dart:112]: Let's create different file for different page

**Review by kovarny — CHANGES_REQUESTED** [review]

**Comment by jomanozgml** [comment]
Different file for different page added

**Review by kovarny — APPROVED** [review]

**Pull request merged**